connector/oidc/oidc.go

```diff
@@ -17,6 +18,11 @@ import (
 	"github.com/dexidp/dex/connector"
 )
 
+type connectorData struct {
+	// Store token, even if it eventually expires
```

Comment:
What happens currently when the token is expired? Do we return nothing? Do we return stale data?